test/script.spec.ts

```diff
@@ -41,6 +41,29 @@ describe('script', () => {
     });
   });
 
+  describe('fromASM', () => {
+    const OPS = bscript.OPS;
+    it('decodes OP_FALSE as empty buffer', () => {
+      const string = 'OP_RETURN OP_FALSE';
+      assert.deepStrictEqual(
+        bscript.fromASM(string),
+        Uint8Array.from([OPS.OP_RETURN, OPS.OP_FALSE]),
+      );
+    });
+
+    it("decodes a series of numbers from '82 to 96' correctly", () => {
```

Comment:
82 to 90 (130 + 14 = 144 = 128 + 16 = 10010000 = 0x90)

Also maybe "hex numbers" or "numbers in hexidecimal"

Thanks for these tests. I think this is the last fix.